CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -1,3 +1,12 @@
+# v1.9.2 - Changelog since v1.9.1
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Revert [#1090](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1090) ([#1174](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1174), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
It would be better if this was more self-descriptive and didn't require someone to click into the PR to understand what changed.

Reply:
/hold